### PR TITLE
connects Falkonry/tracker#616, connects Falkonry/tracker#596

### PR DESCRIPTION
**Pull request by chandresh-gurbani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,6 +279,49 @@ Usage:
...
+```java
+    import com.falkonry.client.Falkonry;
+    import com.falkonry.helper.models.Datasource;
+    import com.falkonry.helper.models.Datastream;
+    import com.falkonry.helper.models.Field;
+    import com.falkonry.helper.models.TimeObject;
+    import com.falkonry.helper.models.Signal;
+
+    //instantiate Falkonry
+    Falkonry falkonry = new Falkonry("https://sandbox.falkonry.ai", "auth-token");
```
Comment (aniket-amrutkar) [inline README.md]: - Change sandbox to localhost in test cases.
- At the top where we fetching logo change sandbox to app

Comment (aniket-amrutkar) [reply README.md]: Make sure that time format is proper at all places after #596 change

File: src/test/java/com/falkonry/TestAddAndGetData.java [hunk 2]
```diff
@@ -20,13 +20,12 @@
...
 import java.util.*;
 
-@Ignore
 public class TestAddAndGetData {
 
 	Falkonry falkonry = null;
 
-	String host = "https://localhost:8080";
-	String token = "lmm3orvm1yaa4j1y5b78i8f870fhon6z";
+	String host = "https://dev.falkonry.ai";
```
Comment (phagunbaya) [inline src/test/java/com/falkonry/TestAddAndGetData.java]: revert this change and remove the token.

File: src/test/java/com/falkonry/TestAddDataStream.java [hunk 3]
```diff
@@ -338,6 +338,61 @@ public void addWideDataCsvStream() throws Exception {
...
+		datastreams.add(datastream);
+
+		Map<String, String> options = new HashMap<String, String>();
+
+		File file = new File("res/data.json");
+		ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(FileUtils.readFileToByteArray(file));
+		InputStatus inputStatus = falkonry.addInputStream(datastream.getId(), byteArrayInputStream, options);
+		Assert.assertEquals(inputStatus.getAction(), "ADD_DATA_DATASTREAM");
+		Assert.assertEquals(inputStatus.getStatus(), "PENDING");
+		Datastream datastream1 = falkonry.getDatastream(datastream.getId());
```
Comment (phagunbaya) [inline src/test/java/com/falkonry/TestAddDataStream.java]: name is as `createDatastream`
